### ISSUE TITLE
Check syntax for asset url in SASS

### ISSUE THREAD

**Issue by LukasKalbertodt** [JulianKniephoff/seminar-manager#26]
We are still not sure if this is correct. https://github.com/JulianKniephoff/seminar-manager/pull/25/files#r52791058

**Comment by JulianKniephoff** [comment]
Right, IMHO this should be `asset-url` or rather even `image-url` instead of `asset_url`.
